packages/teams-js/src/public/monetization.ts

```diff
@@ -1,10 +1,17 @@
-import { sendAndHandleSdkError } from '../internal/communication';
+import { sendAndHandleSdkErrorWithVersion } from '../internal/communication';
 import { ensureInitialized } from '../internal/internalAPIs';
+import { ApiName, ApiVersionNumber, getApiVersionTag } from '../internal/telemetry';
 import { callCallbackWithErrorOrResultOrNullFromPromiseAndReturnPromise, InputFunction } from '../internal/utils';
 import { errorNotSupportedOnPlatform, FrameContexts } from './constants';
 import { SdkError } from './interfaces';
 import { runtime } from './runtime';
 
+/**
```

Comment:
@TrevorJoelHarris , here is the first time I made an update for capability mixing deprecated APIs and new(Beta) APIs in the same file so I just explicitly point it out for you to review.

Reply:
Looks good to me!